tools/ddsrouter_yaml_validator/ddsrouter_yaml_validator/ddsrouter_config_schema.json

```diff
@@ -133,7 +138,14 @@
                 "connection-addresses":{
                     "type":"array",
                     "items":{
-                        "$ref":"#/definitions/ConnectionAddress"
+                        "anyOf":[
+                            {
+                                "$ref":"#/definitions/DsConnectionAddress"
+                            },
+                            {
+                                "$ref":"#/definitions/Address"
+                            }
+                        ]
```

Comment:
Wouldn't this mean that a Discovery Server participant with connection addresses without GUID would be valid (which could be ok to use default value, but we are specifying it to be required in the schema), and an Initial Peers Participant with connection address including GUID would also be fine?

Reply:
It means I did not find a way to do it other way :sweat_smile: